### DISCUSSION TITLE
Searching for the Saas.Notifications Logic App (code or installation in Azure Portal)

### DISCUSSION THREAD

**Discussion by GrahameP42** [Azure/azure-saas#248] [Q&A]
Hi team

This looks to be a great starter kit and the documentation is mostly adequate (and I'll be putting forward some notes from my installation - it's definitely not a 15 minute deployment; I've spent nearly 24 hours so far!)

I have deployed everything to the point of requiring the Saas.Notifications Logic App.  It does not appear to be in my Git code fork at all and there is nothing in my Azure instance under the Logic Apps blade.

Can anyone help me out please?  I'd hate to have come this far and not even be able to register an (admin) account because I can't verify my email?!?!

TIA
Petchy

EDIT:  Interestingly a notification email was received to the subscribing account after about 90 minutes.  The registration portal did not recognize the code and I sent another request for verification code and this one came through immediately to the same address.  Subsequent different addresses appear to have the initial delay.  As yet, none of the verification codes have been accepted and I have yet to succeed registering an account.  I will investigate and update.

**Comment by thorecaspersen** [comment]
Same for me. The docs mention this. But not part of the repo...
